### PR TITLE
Token schema, endpoint for create and read token

### PR DESCRIPTION
**Pull request by angelafeliciaa**

### CODE REVIEW

**Comment by angelafeliciaa** [comment]
Screenshot 2024-10-12 at 14 14 48
Screenshot 2024-10-12 at 14 15 19

File: core/i18nilize/views.py [hunk 1]
```diff
@@ -17,4 +19,33 @@ def post(self, request, *args, **kwargs):
...
+        try:
+            token = Token.objects.get(pk=pk)
+            data = {
+                'id': token.id,
+                'value': str(token.value),
+                'created_at': token.created_at.isoformat()
+            }
+            return Response(data, status=status.HTTP_200_OK)
+        except Token.DoesNotExist:
+            return Response({'error': 'Token not found.'}, status=status.HTTP_404_NOT_FOUND)
```
Comment (kevinrczhang) [inline core/i18nilize/views.py:51]: wrap this into one APIVIew called TokenView, 
get and post should be functions underneath the class

**Review by kevinrczhang — CHANGES_REQUESTED** [review]
Great work! Just a minor suggestion + I would like to see some unit tests implemented through. You can see how they're done here:
https://docs.djangoproject.com/en/5.1/topics/testing/overview/

**Comment by kevinrczhang** [comment]
One more thing, we will want to fetch by token value. Fetching by their primary key is insecure. Otherwise, everything else is great!

**Review by hriday330 — APPROVED** [review]
Unit tests look good! Great work - once Kevin has also approved you can merge it

**Review by kevinrczhang — APPROVED** [review]
:shipit:

**Pull request merged**